### PR TITLE
Minimal documentation for metrics

### PR DESCRIPTION
**Pull request by mosabua**
https://issues.sonatype.org/browse/NEXUS-8507

### CODE REVIEW

File: chapter-configuration.asciidoc [hunk 1]
```diff
@@ -958,12 +958,27 @@ size of the log displayed. A manual refresh can be triggered with the
...
+The 'Metrics' feature view is available in the 'Support' section of
+the 'Administration' main menu. It provides insight to characteristics
+of the Java virtual machine JVM running Nexus and is displayed in
+<<fig-metrics>>.
 
+[[fig-metrics]]
+.JVM Metrics
+image::figs/web/metrics.png[scale=40]
+
+The 'Memory Usage', 'Memory distribution' and 'Thread states' charts
```
Comment (joedragons) [inline chapter-configuration.asciidoc]: PRO shows a lower case U in usage.

**Comment by joedragons** [comment]
+1

**Comment by mosabua** [comment]
http://bamboo.s/browse/BOOK-NEXUS14-1/artifact/STAGE/site/3.0/reference/admin-support.html#metrics

File: chapter-configuration.asciidoc (hunk 1 above)
Comment (mosabua) [inline chapter-configuration.asciidoc]: OSS does to ;-)  

Just habit on my side

**Pull request merged**